e2e/charts/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -38,28 +38,63 @@ spec:
 {{ toYaml .Values.image.pullSecrets | indent 8 }}
       {{- end }}
       containers:
-        - name: {{ template "newrelic.name" . }}
-          image: {{ template "newrelic.image" . }}
-          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        - name: kubelet-scraper
+          image: {{ .Values.scraperImage.repository }}:{{ .Values.scraperImage.tag }}
```

Comment:
```suggestion
          image: {{ .Values.scraperImage.repository }}:{{ .Values.scraperImage.tag | default .Chart.AppVersion }}
```

Reply:
Since all images (except the agent) will come from this repo, it is safe to assume that the default version for all of them could be `Chart.AppVersion`, right?